CASC/Schedules.cpp

```diff
@@ -1953,8 +1953,8 @@ void Schedules::getStructInductionTipSchedule(const Shell::Property& property, S
   // Overall score 235.07056943321408 probs on average / budget 24401 Mi
 }
 
-void Schedules::getSnakeTptpUnsSchedule(const Shell::Property& property, Schedule& quick) {    
-  if (property.hasNumerals() || property.hasInterpretedOperations()) { 
+void Schedules::getSnakeTptpUnsSchedule(const Shell::Property& property, Schedule& quick) {
+  if (property.hasNumerals() || property.hasInterpretedOperations()) {
```

Comment:
@joe-hauns is this the right property invocation?

Reply:
A good question. But I think so. It's the one that worker in the past. 

(But a different one, which perhaps boils down to the same thing on TPTP at the moment, could be more principled. Do you have a suggestion what could be better?)

Reply:
No idea - it just looks a bit fishy, I could imagine:
1. problems which have numerals but only use them as constants
2. problems with interpreted non-arithmetic operations

So I guess really we want `property.hasArithmeticOperations()`?

Reply:
In fact, the whole "TPTP's SPC endswith ARI" is (on v9.0.0) equivalent to current Vampire's `hasNumerals=1`.

Even 
```
tff(a,type,c:$int).
tff(a,axiom,$less(c,c)).
```
triggers `hasNumerals`, I guess we could drop the whole `hasInterpreted` from the disjunction...

Reply:
Probably a job for a different PR, though!